draft-ietf-mls-architecture.md

```diff
@@ -657,14 +644,22 @@ a new client by the protocol. This client will not gain access to the history
 even if it is owned by someone who owns another member of the group.
 Restoring history is typically not allowed at the protocol level but
 applications can elect to provide such a mechanism outside of MLS.
-Such mechanisms, if used, may undermine the FS and PCS guarantees
+Such mechanisms, if used, may reduce the FS and PCS guarantees
 provided by MLS.
 
-### Extensibility / Pluggability
+### Extensibility
+
+The MLS protocol provides several extension points where additional information
+can be provided.  Extensions to KeyPackages allow clients to disclose additional
```

Comment:
It's not clear to me what this first sentence means. Additional information provided where/to whom? Maybe it would make sense to structurally introduce the different extension categories here, i.e. one bullet point per extension category.

Reply:
Looks fine, I think this might be a language thing.